### ISSUE TITLE
Do not hide some tray icons

### ISSUE THREAD

**Issue by Dannniello** [martinpl/tray-icons-reloaded#22]
It is great that Tray-Icons-Reloaded could hide too many tray icons in "..." menu, but some icons should be always visible. For example chat applications like Signal - without visible icon - no visible information that there are unread messages...

It would be excellent if there would be option to choose which icon should be always visible (and/or opposite: which icons should be always hidden in "..." menu).

**Comment by ArjunSingh99** [comment]
I agree, the 3 dot menu should be an additional icon, not a replacement when the space is filled. This could preferably be in the order of which the apps were started.

**Comment by stanfieldr** [comment]
I don't want to hide any icons, but this would help me as well.

**Comment by Asghwor** [comment]
It would be nice if we could decide which icons were hidden and which were not, kinda like how Windows does it.

If drag n' drop is impossible, it could do something similar to the *Collapsible Systray* Cinnamon applet and have us show / hide them with toggles:

![image](https://user-images.githubusercontent.com/30949004/180207631-3bee4223-c842-4149-92e4-f6452cf71f7a.png)

**Comment by elopsod** [comment]
no luck as I see(

**Comment by Knud3** [comment]
As now, icons can be selected `Hidden` in `Applications` tab. It would be nice to have eg. `Exclude from icons limit` or `Unrestricted` toggle switch there too.

**Comment by elkh510** [comment]
Hi @martinpl 
First of all thank you for your work on this project — it’s greatly appreciated!

When you have a chance, could you please take a look this issue
It would be great to hear your thoughts or get any updates.

Thanks again!
